README.md

```diff
@@ -57,3 +57,45 @@ Documentation is lacking at the moment, but contributions are welcome!
 1. Run `cd Example; bundle exec pod install`
 2. Open `Example/KanvasExample.xcworkspace` in Xcode
 3. Run the app on a device.
+
+## Bump version
+
+### Steps
+
+1. Open `Kanvas.podspec`
+2. Upgrade `spec.version`, please refer `Version naming rules` section below.
+3. `cd Example`
+4. Run `pod install`
+5. `cd ..` and update `CHANGELOG.md`
```

Comment:
🔍  Can we add a bit more detail to this step? I believe they have to put any of the unreleased change notes under the current version and check the recent PRs for any merges that have gone in since the last version that haven't made it on to the unreleased section of the change log.